src/HMI/tmplits.js

```diff
@@ -44,7 +44,8 @@ export class Tmplits {
             Tmplits.loadedCallback.push(() => { loadedCallback(this.native) })
         }
         Tmplits.loadedCallback.push(() => { this.refreshDynamicDom() })
-        this.loadPackageLockJson(this.base + '../package-lock.json')
+        this.loadPackageLockJson(this.base + './.package-lock.json')
+            .then(() => { return this.loadPackageLockJson(this.base + '../package.json') })
```

Comment:
2 Questions:

1. Why does loadPackageLockJson always resolve even if it doesnt load?
2. If they are both loaded successfully which one is used?

Reply:
1. Because even it fails, we want to do all the rest.
2. They will both be used. If the content is duplicated, it will use the second one that is loaded

Reply:
1. I get that but the API of loadPackageLockJson feels odd. I am fine with it for now at least